imsv-docs-astro/src/content/docs/guides/core-concepts/card-funding.mdoc

```diff
@@ -42,41 +42,60 @@ of digital assets needed to meet a users desired spend in their desired fiat
 currency. They also provide pre-built `"smart-contract-write"` transactions for
 successful interactions with the Immersve smart contract.
 
+If multiple transactions are present then they should be carried out in the
+order in which they are presented.
 
-### Currency Conversion
+The client application is responsible for creating the raw blockchain transaction
+so that it can be signed and submitted to the relevant blockchain through the
+cardholder's web3 wallet.
 
-A user may be quoted a price for a purchase by a merchant in a local fiat
-currency. To determine the sufficient amount of local fiat currency to fund a
-card in its billing currency (USD), use the [currency
-conversion](/api-reference/currency-conversion) API endpoint.
+<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764577778509789&cot=14 -->
+![Card Funding Sequence](@assets/diagrams/card-funding-sequence.svg)
 
-The returned value can passed to the get spending prerequisites endpoint as
-detailed below.
 
+## Withdrawing Funds
 
-### Lock Funds
+Withdrawals are conducted as a two-stage process being;
 
-To ensure that funds are sufficiently locked within the [smart
-contract](/guides/universal-evm-smart-contract) such that Immersve are in a position to
-approve an authorization request received via the card scheme network you will
-need to have first locked sufficient digital assets within the smart contract
-via digital asset transfer and smart contract invocations as instructed by the
-[get spending prerequisites](/api-reference/get-spending-prerequisites)
-operation. The necessary blockchain transactions are contained within the
-`prerequisites` collection returned.
+1. Obtaining an Immersve-generated signature permitting a given withdrawal
+2. Executing a withdrawal and observing the effect on the funding source
 
-If the user has not transacted using the solution before then the [get spending
-prerequisites](/api-reference/get-spending-prerequisites) response will
-typically call for an ERC20 `approve` in favour of the smart contract followed
-by a `depositAndLock` invocation of the smart contract.
+To begin the withdrawal process your application will
+[create a withdrawal intent](https://docs.immersve.com/api-reference/create-withdrawal-intent/),
```

Comment:
very minor but we probably don't need the full URL here, can use a relative link